genie-web/src/main/resources/db/migration/mysql/V4_0_0_6__Change_Timestamp_Precision.sql

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
No indexing for the new columns. Just checking this is deliberate and not forgotten.
(makes sense to me that we don't need it, at least for the current/expected usage)

Reply:
Yeah we could put one on the `archive_status` field before we forget. Good call.

As for the TEXT/JSON fields. I tried to move some of the columns to the native `json` types in each of the databases we currently support which would allow indexing to make more sense on those fields. However I got into all kinds of problems (most of which I forget at the moment) so I punted and just made them TEXT fields. In mysql at least indexing these makes no sense really. These fields will all hold some arbitrary structured data so unless we enable full text search I doubt we'll be searching through them. They're more likely to be point queries from the app level at least unless you can think fo something else?

Reply:
I agree indexing those JSON fields is probably a waste of resources.